modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -13,6 +13,7 @@ const defaultProps = {
   // TODO - change to onViewportLoad to align with Tile3DLayer
   onViewportLoad: {type: 'function', optional: true, value: null, compare: false},
   onTileLoad: {type: 'function', value: tile => {}, compare: false},
+  onTilePurged: {type: 'function', value: tile => {}, compare: false},
```

Comment:
Not a bad name. I wonder if `onTileUnload` would make sense? It would be more symmetrical with `onTileLoad`...

Reply:
Yea I think `onTileUnload` is a good name for this - I'll make the change.